features/pom.xml

```diff
@@ -91,29 +91,29 @@
         <!-- CXF Guava version must match DDF Guava version to avoid OSGi uses constraint violations -->
         <cxf.guava.version>${guava.version}</cxf.guava.version>
         <cxf.hazelcast.version>${hazelcast.version}</cxf.hazelcast.version>
-        <cxf.hibernate.validator.version>6.1.4.Final</cxf.hibernate.validator.version>
+        <cxf.hibernate.validator.version>6.1.7.Final</cxf.hibernate.validator.version>
         <cxf.httpcomponents.asyncclient.version>4.1.4</cxf.httpcomponents.asyncclient.version>
-        <cxf.httpcomponents.client.version>4.5.12</cxf.httpcomponents.client.version>
+        <cxf.httpcomponents.client.version>4.5.13</cxf.httpcomponents.client.version>
         <cxf.httpcomponents.core.version>${httpcore.version}</cxf.httpcomponents.core.version>
-        <cxf.jackson.databind.version>2.11.2</cxf.jackson.databind.version>
-        <cxf.jackson.version>2.11.2</cxf.jackson.version>
+        <cxf.jackson.version>2.11.4</cxf.jackson.version>
+        <cxf.jackson.databind.version>2.11.4</cxf.jackson.databind.version>
         <!-- Using jakarta.activation 1.2.1 instead of 1.2.2, as it requires Java 9.
              See https://github.com/eclipse-ee4j/jaf/issues/37 -->
         <cxf.jakarta.activation.version>${jakarta.activation.version}</cxf.jakarta.activation.version>
         <cxf.james.mim4j.version>0.7.2</cxf.james.mim4j.version>
-        <cxf.javassist.version>3.25.0-GA</cxf.javassist.version>
+        <cxf.javassist.version>3.27.0-GA</cxf.javassist.version>
         <cxf.javax.annotation-api.version>${javax.annotation-api.version}</cxf.javax.annotation-api.version>
         <cxf.jboss.logging.version>3.3.0.Final</cxf.jboss.logging.version>
         <cxf.jettison.version>1.4.1</cxf.jettison.version>
         <cxf.joda.time.version>${joda-time.version}</cxf.joda.time.version>
-        <cxf.johnzon.version>1.2.8</cxf.johnzon.version>
+        <cxf.johnzon.version>1.2.11</cxf.johnzon.version>
         <cxf.mina.version>2.1.3</cxf.mina.version>
         <cxf.neethi.version>3.1.1</cxf.neethi.version>
-        <cxf.netty.version>4.1.50.Final</cxf.netty.version>
+        <cxf.netty.version>4.1.65.Final</cxf.netty.version>
         <cxf.opensaml.osgi.version>${opensaml.osgi.version}</cxf.opensaml.osgi.version>
         <cxf.opentracing.version>0.33.0</cxf.opentracing.version>
         <cxf.reactivestreams.version>1.0.3</cxf.reactivestreams.version>
-        <!-- Using saaj-impl 1.3.28 instead of 1.5.2, as it requires packages saaj-api 1.4+
+        <!-- Using saaj-impl 1.3.28 instead of 1.5.3, as it requires packages saaj-api 1.4+
```

Comment:
Is this comment still true when upgrading to karaf 4.3.3 ?

Reply:
I think it still applies.

https://github.com/apache/karaf/blob/karaf-4.3.3/bom/pom.xml#L1080-L1084